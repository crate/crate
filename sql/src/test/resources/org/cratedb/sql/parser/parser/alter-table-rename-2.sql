ALTER TABLE t1 RENAME TO t2