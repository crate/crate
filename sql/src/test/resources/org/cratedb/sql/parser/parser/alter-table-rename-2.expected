org.cratedb.sql.parser.parser.AlterTableNode@4dd849ae
name: t1
statementType: ALTER TABLE
objectName: t1
compressTable: false
sequential: false
truncateTable: false
purge: false
defragment: false
truncateEndOfTable: false
updateStatistics: false
updateStatisticsAll: false
indexNameForUpdateStatistics: null
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@32cd666f

	[0]:	
	org.cratedb.sql.parser.parser.AlterTableRenameNode@4b001b77
	name: t2
	elementType: AT_RENAME